FirebaseSwiftUI/FirebaseEmailAuthSwiftUI/Sources/FirebaseEmailAuthSwiftUI/Views/EmailEntry.swift

```diff
@@ -0,0 +1,45 @@
+import SwiftUI
+
+public struct EmailEntryView: View {
+  @State private var email: String = ""
+  @State private var invalidEmailWarning: Bool = false
+  @EnvironmentObject var authFUI: FUIAuth
+
+  public var body: some View {
+    if invalidEmailWarning {
```

Comment:
This will break view identity - see https://swiftwithmajid.com/2021/12/09/structural-identity-in-swiftui/